SimplenoteIntents/IntentHandler.swift

```diff
@@ -0,0 +1,26 @@
+import Intents
+
+class IntentHandler: INExtension {
+    override func handler(for intent: INIntent) -> Any {
+        return self
+    }
+}
+
+extension IntentHandler: SPNoteWidgetIntentHandling {
+    func provideNoteOptionsCollection(for intent: SPNoteWidgetIntent, with completion: @escaping (INObjectCollection<SPNote>?, Error?) -> Void) {
+        // This is placeholder code to confirm the dynamic intent selection is working
+        // TODO: add logic to fetch the available notes for account
+
+        let placeholderNoteData = ["Note 1", "Note 2", "Note 3", "Note 4"]
+
+        var notes: [SPNote] = []
```

Comment:
Can't find where `SPNote` is defined, but can we rename that to `IntentNote` or `WidgetNote` or so?